src/components/Author/index.module.scss

```diff
@@ -0,0 +1,18 @@
+.root {
+  display: flex;
+  align-items: center;
+
+  margin-bottom: 28px;
+
+  @media (min-width: 950px) {
+    margin-bottom: 0;
+  }
```

Comment:
Let's move out layout properties to a layout component or to the "page component" that's holding the `Author` component